src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +31,18 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```

Comment:
Maybe you should consider adding some comments why you want to throw this exception